helm/teleport-kube-agent/templates/service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Release.Name }}
+    {{- include "labels.common" . | nindent 4 }}
+  {{- if .Values.extraLabels.service }}
+    {{- toYaml .Values.extraLabels.service | nindent 4 }}
+  {{- end }}
+  {{- if .Values.annotations.service }}
```

Comment:
Maybe add placeholders for this in `extraLabels` and `annotations` in values.yaml